server/pages/desktop.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { jsonStringifyForHtml } from 'sanitize';
+
+class Desktop extends React.Component {
+	getStylesheet() {
+		const { env, isDebug, isRTL, urls } = this.props;
```

Comment:
Camel-case variable naming applies to acronyms too: `isRTL` should be `isRtl` ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions)).